test/DacpacTool.Tests/ScriptParserTests.cs

```diff
@@ -62,7 +62,7 @@ public void CanParseFileWithIncludesFromRelativePaths()
             // Assert
             result.ShouldNotBeNull();
             result.Count().ShouldBe(2);
-            result.ToList().Take(2).Last().ShouldBe($"{TEST_PROJECT_PATH}/Pre-Deployment/./MoreScripts/Script2.sql");
+            result.ToList().Take(2).Last().Replace("\\", "/").ShouldBe($"{TEST_PROJECT_PATH}/Pre-Deployment/./MoreScripts/Script2.sql");
```

Comment:
Was this change deliberate? This appears to be a platform specific difference, so perhaps we should handle this better?

Reply:
OK. This issue depends on platform that running tests. Windows use `\`, *nix use `/`, Some other magic platform might use `-` for example. IMHO comparing static string with OS results is not a good idea, So I think that it is good place to make refactoring. But for now I only correct line to avoid failing on main platforms (*nix & windows). Editing all unit-tests is not a deal of this PR. I only make editions for tests don't failure. I think that you can do proper changes after that PR.

Reply:
I have Windows 10 Pro 19042.928 installed on my machine. With default Russian locale and Russian language. Also I have language pack installed to .NET SDK. I don't know this is the reason of error or not but the result of test run on picture at attach
![Debug](https://user-images.githubusercontent.com/840112/117937360-907da700-b30e-11eb-85a0-a9586aae8152.png)